src/test/kotlin/no/nav/syfo/integration/kafka/journalpost/JournalpostHendelseConsumerTest.kt

```diff
@@ -0,0 +1,110 @@
+package no.nav.syfo.integration.kafka.journalpost
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import kotlinx.coroutines.runBlocking
+import no.nav.helse.arbeidsgiver.bakgrunnsjobb.BakgrunnsjobbRepository
+import no.nav.syfo.integration.kafka.joarkLocalProperties
+import no.nav.syfo.kafkamottak.InngaaendeJournalpostDTO
+import no.nav.syfo.repository.DuplikatRepository
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class JournalpostHendelseConsumerTest {
+
+    var duplikatRepository: DuplikatRepository = mockk(relaxed = true)
+    var bakgrunnsjobbRepo: BakgrunnsjobbRepository = mockk(relaxed = true)
+    var om: ObjectMapper = mockk(relaxed = true)
+
+    var props = joarkLocalProperties().toMap()
+    val topicName = "topic"
+    val GYLDIG_INNTEKTSMELDING = InngaaendeJournalpostDTO("abc", 1, "", 111, "MOTTATT", "", "SYK", "ALTINN", "", "")
+    val DUPLIKAT_INNTEKTSMELDING = GYLDIG_INNTEKTSMELDING.copy(journalpostId = 222)
+    val IKKE_INNTEKTSMELDING = InngaaendeJournalpostDTO("abc", 1, "", 333, "IKKE_MOTTATT", "", "IKKE_SYK", "IKKE_ALTINN", "", "")
+
+    @Test
+    fun isready_skal_gi_feilmelding_før_oppstart() {
+        val consumer = JournalpostHendelseConsumer(props, topicName, duplikatRepository, bakgrunnsjobbRepo, om)
```

Comment:
Lage en `@BeforeEach` for opprettelse av consumer?

Reply:
Ja. Fikser